app/lib/services/feature.rb

```diff
@@ -2,14 +2,10 @@ module Services
   class Feature
     REGISTRATION_OPEN_DATE = Time.zone.parse("6 June 2022 12:00")
 
-    GAI_INTEGRATION_KEY = "Get an Identity integration".freeze
```

Comment:
This is already always at 100%, this change is here since the code now always has it turned on and users can't be "removed from the pilot" anymore since the non-pilot journey has been removed and replaced with the signle qualified_teacher_check question.